paasta_tools/paasta_metastatus.py

```diff
@@ -601,7 +619,7 @@ def main():
         mesos_state=mesos_state,
     )
     metrics = get_mesos_stats()
-    mesos_metrics_status = get_mesos_metrics_health(mesos_metrics=metrics)
+    mesos_metrics_status = get_mesos_metrics_health(mesos_metrics=metrics, mesos_state=mesos_state)
```

Comment:
sure, but we can't really rely on MESOS-6175. I absolutely agree that we should do this in parallel - in fact, I deleted an unused function from mesos-cli that did this. We can always bring that back to life. Let's see how this goes - I have a feeling we're going to suffer from these calls to the master state.json - but we can ship this version for now